front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -91,9 +91,10 @@ public StorageServiceSupport(ObjectType objectType,
       this, new ToDoubleFunction() {
         @Override
         public double applyAsDouble(Object ignore) {
-          return allItemsCache.get().size();
+          Set itemCache = allItemsCache.get();
+          return itemCache != null ? itemCache.size() : 0;
```

Comment:
Ran into a situation where this NPE'd because the cache hadn't been populated yet.